docs/pages/database-access/guides/dynamodb.mdx

```diff
@@ -0,0 +1,308 @@
+---
+title: Database Access with AWS DynamoDB
+description: How to access AWS DynamoDB with Teleport.
+---
+
+<Admonition type="note" title="Important">
+Access to AWS DynamoDB is provided by [**Teleport Application Access**](../../application-access/introduction.mdx) for AWS Console and API.
+</Admonition>
+
+This guide will help you to:
+- Install the Teleport Application Service.
+- Set up Teleport to access the AWS Console and API.
+- Connect to your DynamoDB databases through Teleport.
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+![DynamoDB Self-Hosted](../../../img/database-access/guides/dynamodb_selfhosted.png)
+</ScopedBlock>
+
+<ScopedBlock scope={["cloud"]}>
+![DynamoDB Cloud](../../../img/database-access/guides/dynamodb_cloud.png)
+</ScopedBlock>
+
+## Prerequisites
+
+- AWS account with DynamoDB databases.
+- IAM permissions to create IAM roles.
+- `aws` (AWS Command Line Interface) installed and added to your system's
+  `PATH` environment variable.
+- A host, e.g., an EC2 instance, where you will run the Teleport Application
+  Service.
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+(!docs/pages/includes/tctl.mdx!)
+
+<Admonition type="tip" title="Not yet a Teleport user?">
+If you have not yet deployed the Auth Service and Proxy Service, you should follow one of our [getting started guides](../getting-started.mdx) or try our Teleport Application Access [interactive learning track](https://play.instruqt.com/teleport/invite/rgvuva4gzkon).
+</Admonition>
+
+We will assume your Teleport cluster is accessible at `teleport.example.com` and `*.teleport.example.com`. You can substitute the address of your Teleport Proxy Service. (For Teleport Cloud customers, this will be similar to `mytenant.teleport.sh`.)
+
+<Admonition type="note" title="Application Access and DNS">
```

Comment:
```suggestion
<Admonition type="note" title="Application Access and DNS" scope={["oss", "enterprise"]} scopeOnly>
```
Suggested hiding this for Cloud users so they don't create unnecessary DNS records.